### ISSUE TITLE
[Website][Doc] Gluon KL divergence link rendering issue

### ISSUE THREAD

**Issue by ehsanmok** [apache/incubator-mxnet#17423]
https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/loss/kl_divergence.html#Kullback-Leibler-(KL)-Divergence

<img width="803" alt="Screen Shot 2020-01-23 at 12 09 50 PM" src="https://user-images.githubusercontent.com/6980212/73019894-59157f00-3dd9-11ea-814d-2e76706081a9.png">

and 

<img width="844" alt="Screen Shot 2020-01-23 at 12 11 32 PM" src="https://user-images.githubusercontent.com/6980212/73019997-9548df80-3dd9-11ea-9208-dbcaefe2e76b.png">

- https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/loss/kl_divergence.html#from_logits=False

<img width="838" alt="Screen Shot 2020-01-23 at 12 12 30 PM" src="https://user-images.githubusercontent.com/6980212/73020051-b0b3ea80-3dd9-11ea-8936-feb592fbbb36.png">

- https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/loss/kl_divergence.html#Advanced:-Common-Support

<img width="809" alt="Screen Shot 2020-01-23 at 12 13 33 PM" src="https://user-images.githubusercontent.com/6980212/73020137-dc36d500-3dd9-11ea-8db5-515f6819b7a8.png">

- https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/loss/kl_divergence.html#Advanced:-Aggregation

<img width="811" alt="Screen Shot 2020-01-23 at 12 14 27 PM" src="https://user-images.githubusercontent.com/6980212/73020184-f670b300-3dd9-11ea-9808-4fe496561c30.png">

**Comment by ehsanmok** [comment]
@mxnet-label-bot add [Doc, Good First Issue]

**Comment by TEChopra1000** [comment]
This issue was resolved and this ticket can be closed.

**Comment by TEChopra1000** [comment]
PR: https://github.com/apache/incubator-mxnet/pull/17435
